### PR TITLE
Fix AMD shader compilation

### PR DESCRIPTION
**Pull request by SamGondelman**
This should fix some shader compilation issues on AMD cards.

Test plan:
- Run Interface on: Windows (NVIDIA and AMD cards), mac, and android.  Check logs for shader compilation errors.  You shouldn't see any.

### CODE REVIEW

File: libraries/graphics/src/graphics/Material.slh [hunk 1]
```diff
@@ -13,14 +13,11 @@
 
 // The material values (at least the material key) must be precisely bitwise accurate
 // to what is provided by the uniform buffer, or the material key has the wrong bits
-#ifdef GL_ES
```
Comment (SamGondelman) [inline libraries/graphics/src/graphics/Material.slh]: (This was the problem, missing #ifndef PRECISIONQ here)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8862/app_12921_PR_9ab841d3548b8185f3d9ef0dc87f3c83f7b4534a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8862/HighFidelity-Beta-PR12921-9ab841d3548b8185f3d9ef0dc87f3c83f7b4534a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8862/HighFidelity-Beta-Interface-PR12921-9ab841d3548b8185f3d9ef0dc87f3c83f7b4534a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8862/HighFidelity-Beta-PR12921-9ab841d3548b8185f3d9ef0dc87f3c83f7b4534a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8862/HighFidelity-Beta-Interface-PR12921-9ab841d3548b8185f3d9ef0dc87f3c83f7b4534a.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8876/app_12921_PR_901301485b3b694c674d8545e13df9bd8dd21ffb.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8876/HighFidelity-Beta-PR12921-901301485b3b694c674d8545e13df9bd8dd21ffb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8876/HighFidelity-Beta-Interface-PR12921-901301485b3b694c674d8545e13df9bd8dd21ffb.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8876/HighFidelity-Beta-PR12921-901301485b3b694c674d8545e13df9bd8dd21ffb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8876/HighFidelity-Beta-Interface-PR12921-901301485b3b694c674d8545e13df9bd8dd21ffb.exe

**Comment by L370** [comment]
@Zvork @NissimHadar For HMD testing it seems that we will need some manual intervention to get the script to function correctly, yes? Regarding the various GPU's that all tests should be run against, I can get a list from our QA team of all systems we can run these tests against and build a matrix in testrail to qualify the various gpu/processor combos.

**Comment by samcake** [comment]
IN this particular instance, the GPU matter, we absolutely want to test on NVIDIA and AMD GPU
OS should be tested on window and macos
HMD is not a differentiation so only Desktop can be tested.

**Comment by NissimHadar** [comment]
In general, there is no good way to automate HMD tests.  There is usually
no image in an HMD if it is not being worn, and if it is being worn then
the image won't be static.

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Only warning seen was on the MinSpec machine: WARNING: Too many temp register is used in Vertex shader, it may cause slow execution. This is not a compilation error however so this is still a pass. Tested on AMD and Nvidia Windows and Mac with AMD.

**Pull request merged**